tests/test_grains.py

```diff
@@ -13,6 +14,61 @@
 TOLERANCE = {"atol": 1e-07, "rtol": 1e-07}
 
 
+grain_array = np.array(
+    [
+        [0,0,1,1,1,1,1,0,0,0],
+        [0,1,1,1,1,0,1,0,0,2],
+        [0,0,0,0,0,0,0,0,0,2],
+        [0,3,3,0,0,0,0,0,2,2],
+        [3,3,3,3,3,0,0,2,2,2],
+    ]
+)
+
+grain_array2 = np.array(
+    [
+        [0,0,0,0,0,0,0,0,0,0],
+        [0,0,0,0,0,0,0,0,0,1],
+        [0,0,0,0,0,0,0,0,0,1],
+        [0,2,2,0,0,0,0,0,1,1],
+        [2,2,2,2,2,0,0,1,1,1],
+    ]
+)
+
+grain_array3 = np.array(
+    [
+        [0,0,0,0,0,0,0,0,0,0],
+        [0,0,0,0,0,0,0,0,0,0],
+        [0,0,0,0,0,0,0,0,0,0],
+        [0,0,0,0,0,0,0,0,0,0],
+        [0,0,0,0,0,0,0,0,0,0],
+    ]
+)
+
+grain_array4 = np.array(
+    [
+        [0,0,1,1,1,1,1,0,0,0],
+        [0,1,1,1,1,0,1,0,0,0],
+        [0,0,0,0,0,0,0,0,0,0],
+        [0,0,0,0,0,0,0,0,0,0],
+        [0,0,0,0,0,0,0,0,0,0],
+    ]
+)
+
+@pytest.mark.parametrize(
+    "area_thresh, expected",
+    [
+        ([None, None], grain_array),
+        ([None, 8], grain_array2),
+        ([3,6], grain_array3),
+        ([8,11], grain_array4)
+    ]
+)
+def test_known_array_threshold(area_thresh, expected) -> None:
+    "Tests that arrays the thresholded on size as expected"
```

Comment:
Sorry another pedantic typo
```suggestion
    "Tests that arrays are thresholded on size as expected."
```